client/src/components/UI/Toast.tsx

```diff
@@ -0,0 +1,58 @@
+import React,{useCallback, useEffect} from "react";
+import { toastActions } from "../../store/toast";
+import { useDispatch } from "react-redux";
+import styles from "./Toast.module.css";
+
+export interface IToastProps {
+    id: number;
```

Comment:
check indentation,. for this file it looks like 4 spaces,. but for other its 2 spaces